### PR TITLE
Bumping CP Halo Windows version to 4.2.7

### PR DESCRIPTION
**Pull request by sostingy**
Eric Abrams has requested that we update the version of the CP Windows installer to 4.2.7 to reflect the update on https://one.rackspace.com/display/SPA/RMS+PDR+Supported+Operating+Systems+Managed+Security

I'm not sure what else is required to get this deployed, but this seem similiar to the request I made in RackSec/infrastructure#145.

### CODE REVIEW

**Comment by stephenl03** [comment]
I submitted an issue to Cloudpassage to create a symlink from latest to the latest version. We will see if they will accommodate.

**Comment by stephenl03** [comment]
Just to follow-up on this item. Cloudpassage responded with...

It is possible to programmatically obtain the URL for the latest Windows Halo Agent via the Halo API. The following API call will output the unattended installation script for Windows. You can then parse out the URL that follows the "-source" parameter.

GET https://api.cloudpassage.com/v2/installers?os=windows&type=agent&distribution_type=software

Example output:
{
script: "#\n#  CloudPassage Halo Agent\n#  Unattended installation script for Windows\n#  -------------------------------------------------------------------------\n#  This script is intended to be used for an unattended installation\n#  of the CloudPassage Halo agent.\n#\n#  IMPORTANT NOTES\n#\n#    * This script is designed to be run in Windows PowerShell 3.0 and later.\n#    * This script assumes that Microsoft BITS is installed and enabled.\n#    * This script may require adjustment to conform to your server\n#      configuration. Please review this script and test it on a server\n#      before using it to install the Halo agent on multiple servers.\n#    * This script contains the CloudPassage Halo agent registration key.\n#      Keep this script safe - handle it as you would the password to\n#      your CloudPassage portal account!\n#\n# download Halo agent installer\nstart-bitstransfer -source https://production.packages.cloudpassage.com/windows/cphalo-4.2.7-win64.exe\n\n# install and run agent for the first time\n.\\cphalo-4.2.7-win64.exe --% /S /agent-key=xxxxxx /grid=\"https://grid.cloudpassage.com/grid\"\n\n# remove installer\nerase .\\cphalo-4.2.7-win64.exe\n"
}

TL;DR
Use their API to pull the install script to get the latest version of the agent.
